test/fluree/db/query/fql_test.clj

```diff
@@ -0,0 +1,29 @@
+(ns fluree.db.query.fql-test
```

Comment:
Does it make sense to test this in CLJS too?

Reply:
> ... a couple questions about testing in CLJS ...
Funny you should mention that. These are actually new test files in these branches. I reflexively added them as cljc files, and they fail on the cljs side. 

The other newly added json-ld tests are all clj only, so I decided not to let the cljs failures hold back this merge because there's a lot of code here, so I decided to make those new files clj only, and I'm planning on opening another issue to make the cljs tests work for all the new json-ld stuff too.